W-10673838 veeva vault clean up (#1802)
mulesoft/docs-connectors: veevavault/1.3/modules/ROOT/pages/index.adoc

Anypoint Connector for Veeva Vault (Veeva Vault Connector) enables you to connect Mule flows to the Veeva Vault platform using the Veeva Vault REST API.

> This connector works with the https://developer.veevavault.com/api/19.1/#authentication[Veeva Vault REST API, Bulk API, and Streaming API], depending on the operations you configure. Each API call uses a JSON request or a CSV request and returns JSON input stream responses over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.

Reviewer: This is completely subjective, but I can more easily understand the first sentence if the modifier comes first:
```suggestion
Depending on the operations you configure, this connector works with the https://developer.veevavault.com/api/19.1/#authentication[Veeva Vault REST, bulk, and streaming APIs]. Each API call uses a JSON request or a CSV request and returns JSON input stream responses over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
```

For information about compatibility and fixed issues, see the Veeva Vault Connector Release Notes.

> == Prerequisites

Reviewer: Is the connectors template being modified to match the team templates?
```suggestion
== Before You Begin
```

To use this connector, you must be familiar with:

* http://vaulthelp2.vod309.com/wordpress/vault-basics/[Veeva Vault]
* https://developer.veevavault.com/vql/#introduction-to-vault-queries[Vault Query Language (VQL)]
* Anypoint Connectors
* Mule runtime engine (Mule)
* Elements and global elements in a Mule flow
* How to create a Mule app using Anypoint Studio

Before creating an app, you must have:

* Credentials to access the Veeva Vault target resource
* Anypoint Platform
* Anypoint Studio version 7.3.5 or later

> == Common Use Cases For the Connector

Reviewer: ```suggestion
== Common Use Cases for the Connector
```
https://capitalizemytitle.com/, *Chicago* tab

Create, retrieve, delete
* Vault Object +
Create, retrieve, update, delete
* Picklists +
Retrieve
* Audit Trail +
Retrieve
* Query +
VQL query to query on documents, Vault objects, and workflows
* Spark validator +

> Validates and verifies the Spark message with a signature that is triggered from Veeva Vault

Reviewer: The verb use is different here (validates, verifies) than in the peer list entries (retrieve, no verb "VQL query"): can you make these texts parallel?

* Invoke Rest API +

> Invokes https://developer.veevavault.com/api/20.1/#documents[Veeva Vault Rest APIs], in which a specific connector operation does not already exist. To maintain https://developer.veevavault.com/docs/#api-rate-limits[API rate limits], use Bulk APIs instead of single APIs.

Reviewer: Are we saying that ALL VV REST APIs do not already have connector operations (comma) or only those that do not (no comma)?
```suggestion
Invokes https://developer.veevavault.com/api/20.1/#documents[Veeva Vault Rest APIs] in which a specific connector operation does not already exist. To maintain https://developer.veevavault.com/docs/#api-rate-limits[API rate limits], use bulk APIs instead of single APIs.
```

== Connection Types

Veeva Vault connections use the Basic authentication type, which uses credentials to connect to a Veeva Vault instance and perform operations.

For information about configuring authentication types, see xref:veevavault-connector-studio.adoc#configure-global-element[Configure a Global Element for the Connector].

== Exchange Templates and Examples

https://www.mulesoft.com/exchange/[Anypoint Exchange] provides templates

> that you can use as a starting point for your app, as well as examples that illustrate a complete solution.

Reviewer: ```suggestion
that you can use as starting points for your apps, as well as examples that illustrate a complete solution.
```
